scripts/gha/print_matrix_configuration.py

```diff
@@ -166,7 +166,7 @@
 #   gcloud firebase test ios models list
 TEST_DEVICES = {
   "android_min": {"type": "real", "device": "model=Nexus10,version=19"},
-  "android_target": {"type": "real", "device": "model=gts4lltevzw,version=28"},
+  "android_target": {"type": "real", "device": "model=blueline,version=28"},
```

Comment:
Just making sure that `blueline` is still version 28.

Reply:
MODEL_ID: blueline
MAKE: Google
MODEL_NAME: Pixel 3
FORM: PHYSICAL
RESOLUTION: 2160 x 1080
OS_VERSION_IDS: 28
TAGS: